app/src/hooks/api/useAlertApi.ts

```diff
@@ -0,0 +1,88 @@
+import { AxiosInstance } from 'axios';
+import {
+  IAlert,
+  IAlertCreateObject,
+  IAlertFilterParams,
+  IAlertUpdateObject,
+  IGetAlertsResponse
+} from 'interfaces/useAlertApi.interface';
+import qs from 'qs';
+
+/**
+ * Returns a set of supported api methods for managing alerts
+ *
+ * @param {AxiosInstance} axios
+ * @return {*} object whose properties are supported api methods.
+ */
+export const useAlertApi = (axios: AxiosInstance) => {
+  /**
+   * Get project alert details based on its ID for viewing purposes.
```

Comment:
nit: Doc seems to be out of date.

Reply:
All of the doc strings seem slightly off. They all mention 'project' even though the alerts arent project specific.

Reply:
Fixed